src/fastfield/serializer/gcd.rs

```diff
@@ -0,0 +1,160 @@
+use fastdivide::DividerU64;
+use gcd::Gcd;
+
+pub const GCD_DEFAULT: u64 = 1;
+
+// Find GCD for iterator of numbers
+pub fn find_gcd(numbers: impl Iterator<Item = u64>) -> Option<u64> {
+    let mut numbers = numbers.filter(|n| *n != 0);
+    let mut gcd = (numbers.next()?).gcd(numbers.next()?);
```

Comment:
`gcd = numbers.next()?;`

is sufficient

actually the current version is incorrect on `0, 10` for instance.